# Specify motor port. 0: Stepper 1; 1: Stepper 2.
uint8 index
# Speed control enable: 1: enable.
uint8 isEnabled
# Speed.
float32 speed
bool isQueued
---
int32 result
uint64 queuedCmdIndex
